buildSrc/src/main/kotlin/Dependencies.kt

```diff
@@ -192,6 +193,9 @@ object Dependencies {
         const val KMS = "com.amazonaws:aws-java-sdk-kms"
         const val SECRETS_MANAGER = "com.amazonaws:aws-java-sdk-secretsmanager"
         const val JAXB = "javax.xml.bind:jaxb-api:$JAXB_VERSION"
+
+        const val BOM2 = "software.amazon.awssdk:bom:$VERSION2"
```

Comment:
Not urgent, but I'll work to update the AWS dependencies to SDKv2 over the next couple of weeks